Source/Objects/movementobject.cpp

```diff
@@ -232,7 +232,7 @@ void MovementObject::Reload() {
     as_context->Reload();
 }
 
-const float kCullRadius = 2.0f;
+float kCullRadius = 2.0f;
```

Comment:
This is a global variable, so if it's changed inside one object, it's changed for every single object being drawn. I don't think that's the right solution. You could copy the constant's value to a local variable inside the function(s) it's used in, or you could make it an member variable of the movement object class, so that it is instanced per object.